server/server.js

```diff
@@ -1,4 +1,8 @@
 const express = require('express');
+
+const server = express();
+const httpServer = require('http').Server(server);
+const io = require('socket.io')(httpServer);
 const next = require('next');
 const bodyParser = require('body-parser');
 const mongoose = require('mongoose');
```

Comment:
consult the coding conventions on how to order these imports

Reply:
@enyeInc I separated the require from initialization, is that ok? I couldnt find anything in the doc on this. And the doc is more inclined to the frontend as opposed to backend
```
const express = require('express');
const http = require('http');
const socketio = require('socket.io');
const next = require('next');
const bodyParser = require('body-parser');
const mongoose = require('mongoose');
const morgan = require('morgan');
const passport = require('passport');
const cookieParser = require('cookie-parser');
const session = require('express-session');
const dotenv = require('dotenv');
const MongoStore = require('connect-mongo')(session);
const path = require('path');
const userInViews = require('./lib/middlewares/userInViews');
const routes = require('./routes');
```

Reply:
i added a comment on how to address this